app/styles/power-select.less

```diff
@@ -1,6 +1,14 @@
 .upf-power-select {
   height: 100%;
 
+  &--error {
+    border: red !important;
```

Comment:
Do you really need the important here and can you use css variable? (is it old code?)

Reply:
I was messing around with the values and forgot about this one, thanks for finding it ! 🙏